package.json

```diff
@@ -85,6 +84,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.8.1",
     "@fortawesome/react-fontawesome": "^0.1.4",
     "classnames": "^2.2.5",
+    "node-sass": "^5.0.0",
```

Comment:
node-sass is actually now deprecated in favor of using Dart sass.

![image](https://user-images.githubusercontent.com/2828721/108372914-4d0f1300-71cd-11eb-920b-5c066de17dee.png)

We updated frontend-build to use Dart sass a few months back: See https://github.com/edx/frontend-build/pull/113 and https://github.com/edx/frontend-build/issues/111.

We'll likely want to make a similar change to Paragon.

Reply:
Per conversation Lael and I had at standup: 1. If Dart sass works, great 2. If Dart sass doesnt work out of box for this, I am strongly in favor of webpack (even if it seems overkill) compared to any completely-new-to-edX alternatives because I do not want us to have 1 repo using a compiler technology: that introduces tooling debt at the organizational level.

Reply:
Dart-sass did work, and I brought the build issue up at the fedex meeting and David Joy alerted me to a different webpack config that's available via frontend-build that we can use to build this. Gonna open a ticket for that.